sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/models/webkey/JsonWebKey.java

```diff
@@ -201,7 +201,7 @@ public JsonWebKey setKeyType(KeyType keyType) {
      */
     @JsonProperty("key_ops")
     public List<KeyOperation> getKeyOps() {
-        return Collections.unmodifiableList(this.keyOps);
+        return this.keyOps == null ? this.keyOps : Collections.unmodifiableList(this.keyOps);
```

Comment:
Is there a difference between a null and an empty list? If not, I would suggest always returns an empty list rather than null.

Reply:
made it return unmodifiable empty list, that would be better